README.md

```diff
@@ -270,11 +264,18 @@ Zarf is written entirely in [go](https://go.dev/), except for a single 400Kb bin
 - Currently the Registry and Git servers _are not HA_, see [#375](https://github.com/defenseunicorns/zarf/issues/376) and [#376](https://github.com/defenseunicorns/zarf/issues/376) for discussion on this
 - In order to avoid TLS issues, Zarf binds to `127.0.0.1:319999` on each node as a [NodePort](https://kubernetes.io/docs/concepts/services-networking/service/#type-nodeport) to allow all nodes to access the pod(s) in the cluster
 - Until [#306](https://github.com/defenseunicorns/zarf/pull/306) is merged, during helm install/upgrade a [Helm PostRender](https://helm.sh/docs/topics/advanced/#post-rendering) function is called to mutate images and [ImagePullSecrets](https://kubernetes.io/docs/concepts/containers/images/#specifying-imagepullsecrets-on-a-pod) so the deployed resoures use the NodePort binding
-- Once [#329](https://github.com/defenseunicorns/zarf/pull/329) is merged, Zarf will use a new injector system to bootstrap a new cluster:
+- With [#329](https://github.com/defenseunicorns/zarf/pull/329), Zarf uses a new injector system to bootstrap a new cluster:
   - Get a list images in the cluster
   - Attempt to create an ephemeral pod using an image from the lsit
   - A small rust binary that is compiled using [musl](https://www.musl-libc.org/) to keep the size the max binary size of ~ 672 KBs is injected into the pod
   - The mini zarf registry binary and `docker:2` images are put in a tar archive and split into 512 KB chunks, larger sizes tended to cause latency issues on low-resource control planes
   - An init container runs the rust binary to reassabmle and extract the zarf binary and registry image
   - The container then starts and runs the zarf binary to host the registry image in an embedded docker registry
-  - After this the main docker registry chart is deployed, pulls the image from the ephemeral pod and destroys the created configmaps, pod and service
\ No newline at end of file
+  - After this the main docker registry chart is deployed, pulls the image from the ephemeral pod and destroys the created configmaps, pod and service
+
+&nbsp;
+### Zarf Architecture
+![Architecture Diagram](./.images/architecture.svg)
+
+
+[Full SVG](./.images/architecture.svg) | [Source DrawIO](docs/zarf-injector.drawio)
```

Comment:
@Madeline-UX should weigh in but would it be better to move the `Zarf-Managed Deployments` underneath the other other boxed diagram? 

Having the arrows leaving Zarf ordered '5,1,2,3,4' doesn't seem as clear at first glance as it could be.

Reply:
Yeah I had it that way initially but it makes giant loops around zarf or over across several other lines and looked really messy. They're also very clearly numbered and that one spaced outside the list visual cue so I think it's fine for now.